disperser/dataapi/docs/swagger.yaml

```diff
@@ -1,5 +1,5 @@
 definitions:
-  core.BlobCommitments:
+  encoding.BlobCommitments:
     properties:
       commitment:
         $ref: '#/definitions/core.Commitment'
```

Comment:
not familiar with swagger, but should the ref 3 lines below be changed too?

Reply:
@jianoaix

Reply:
It can be generated after fix the code: https://github.com/Layr-Labs/eigenda/blob/3c66c994e6ffb09afcf957b68fe6e01d34d614bf/disperser/dataapi/server.go#L42

Reply:
@jianoaix, does anything need to be fixed here before this is merged?

Reply:
Looks good

Reply:
The `core.Commitment` is obsolete but it's since a previous PR. So sounds fine to fix that separately to get this merged now to avoid conflicts (give its size).